db/system_keyspace.cc

```diff
@@ -1933,6 +2028,7 @@ std::vector<schema_ptr> system_keyspace::all_tables(const db::config& cfg) {
                     v3::views_builds_in_progress(), v3::built_views(),
                     v3::scylla_views_builds_in_progress(),
                     v3::truncated(),
+                    v3::commitlog_cleanups(),
```

Comment:
What about extra durability using the metadata commitlog? Also need to ensure this table gets replayed before the data commitlog.

Reply:
Yes, this table of course should be using the schema commitlog. I didn't add it to the relevant list only because I thought it was the default for system tables. I will do that in the next revision.

Reply:
We need a better way to specify that.